internal/cmdapply/cmdapply.go

```diff
@@ -83,6 +84,7 @@ func NewRunner(ctx context.Context, factory util.Factory,
 		"dry-run apply for the resources in the package.")
 	c.Flags().BoolVar(&r.printStatusEvents, "show-status-events", false,
 		"Print status events (always enabled for table output)")
+	c.Flags().StringVar(&r.rgFile, "rg-file", rgfilev1alpha1.RGFileName, "The name of the ResourceGroup file for storing inventory information")
```

Comment:
nit: "the file path to the ResourceGroup object". qq: Shall we mention inventory here or only ResourceGroup?

Reply:
Agree we should rephrase this a bit.

Reply:
Since we still mention inventory elsewhere, I'm leaning more towards maintaining that vocabulary. This is especially since we still support the inventory being provided in the Kptfile - we just throw a warning message encouraging users to migrate to a resourcegroup file instead. But yes, I'll need to update the help message regardless. Lmk what your preferences are in how this should be rephrased.

Reply:
I actually has a strong preference to not surface "inventory" in the resourcegroup case. One of the motivations for the `kpt live apply` change is that we got many feedback the "inventory" "resourceGroup" confuse users. It's harder to let them realize the value of `kpt live apply`.

As for your concern about  "we still support the inventory being provided in the Kptfile", thinking from the users' perspective, they use either inventory or resourcegroup, but not using them at the same time (except migration).

Reply:
Sure - there needs to be a time to make the hard switch anyway, might as well make it now. Have updated the `--rg-file` help message, and other log output to not refer resourcegroup as inventory.